Can't resolve 'process/browser' when using `@codemirror/lang-xml`

I'm struggling with an error with the library `@codemirror/lang-xml`. After I add it to my visual, I start receiving this error during compilation:

```
ERROR in ./node_modules/@lezer/lr/dist/index.js 1039:23-30
Module not found: Error: Can't resolve 'process/browser' in '<PATH>/node_modules/@lezer/lr/dist'
Did you mean 'browser.js'?
BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

I found several solutions that involve adding a custom configuration to webpack.config.js, which we can't do, right?
Do you have any suggestions?
Thanks!

--- a/Changelog.md
+++ b/Changelog.md
@@ -2,6 +2,9 @@
 
 This page contains information about changes to the PowerBI Visual Tools (pbiviz).
 
+## 6.1.2
+* Fixed issue with `process/browser` not being resolved correctly in webpack config
+
 ## 6.1.1
 * Fixed bug with source maps generation (now using `eval-source-map` instead of `source-map`)
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-tools",
-  "version": "6.1.1",
+  "version": "6.1.2",
   "description": "Command line tool for creating and publishing visuals for Power BI",
   "main": "./bin/pbiviz.js",
   "type": "module",

--- a/src/webpack.config.ts
+++ b/src/webpack.config.ts
@@ -38,6 +38,12 @@ const webpackConfig = {
                 }
             },
             {
+                test: /\.m?js/,
+                resolve: {
+                    fullySpecified: false
+                }
+            },
+            {
                 test: /\.json$/,
                 loader: "json-loader",
                 type: "javascript/auto"

